src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -169,8 +183,11 @@ export class BucketDetailComponent implements OnInit {
                   folderContain = true;
                 }else if(ObjectKeyNum == folderNum + 1){
                   //Identify folders within folders
+                  alldir 
```

Comment:
@yuejuSDS @jimccfun Please check and remove this.

Reply:
@anvithks I have remove it, plz check again,thank you.